src/monaco/browser/monaco-frontend-module.ts

```diff
@@ -20,13 +20,16 @@ import { MonacoEditorService } from "./monaco-editor-service";
 import { MonacoModelResolver } from "./monaco-model-resolver";
 import { MonacoContextMenuService } from "./monaco-context-menu";
 import { MonacoCommandService, MonacoCommandServiceFactory } from './monaco-command-service';
+import { bindEditorPreferences } from '../../editor/browser/editor-preferences'
```

Comment:
it should be bound in the editor extension, the monaco extension assume that it is bound.